guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -13,6 +13,13 @@ endif::[]
 .Procedure
 . Create a backup.
 . Restore the backup on the new server.
+. Correct the permissions on the `evr` extension:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# runuser -l postgres -c \
+"psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
```

Comment:
```suggestion
"psql -d foreman -c \
\"UPDATE pg_extension \
SET extowner = (SELECT oid FROM pg_authid \
WHERE rolname='foreman') \
WHERE extname='evr';\""
```
This should still work, right?

Reply:
I guess? :)
I was just copying from #3167

Reply:
Let's stay consistent with #3167 then.

Reply:
In Satellite downstream pdfs, the line gets wrapped like this (example from RHEL docs):

6.2.3. Scanning remote systems for vulnerabilities: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html-single/security_hardening/index#scanning-remote-systems-for-vulnerabilities_vulnerability-scanning (see the two commands under `Procedure`).

Compare that with 6.2.3. Scanning remote systems for vulnerabilities in the pdf build: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/pdf/security_hardening/Red_Hat_Enterprise_Linux-8-Security_hardening-en-US.pdf

Not great, not terrible. As a reviewer, I'll be fine with either.